### DISCUSSION TITLE
MONAI Pathology Labeling  + Active Learning

### DISCUSSION THREAD

**Discussion by Suchi97** [Project-MONAI/MONAILabel#903] [Q&A] [answered]
Hi Team, 

We had attended the MONAI Pathology session conducted on July 28. That was a great session.
And thanks a lot for the information.

In the section of MONAI Label for Pathology, we understood that QuPath, DSA and CVAT  applications are used to annotate the labels for pathology.

In one of the slides, we saw that the deepedit app which is used for the implementation of active learning.

So, Is active learning strategy implemented for annotating the pathology labels?

Also can I use the deepedit app in any of the applications given above

Any help is appreciated.
Thanks in advance

**Comment by SachidanandAlle** [comment, marked as answer]
you can use deepedit model to say segment generic nuclei.. 

but using it for active learning in pathology is more challenging..
may be there is a confusion in understanding.. we only support continuous learning (segment, train, retrain your mode) for pathology example..  for active learning (which samples to pick up out of very larger images.. or which regions to annotate among 150k x 150x image) for pathology is still a challenging one.  happy to see more researches work on this challenge for pathology and contribute the same for pathology app in monailabel.

deepedit training is a simple concept.. you can use the same in solving similar use-case.  but i would say, you need to train enough, so that it can learn both auto-segmentation and interactive-segmentation (currently the split is 50:50) correctly.. balancing between those 2 samples inputs and getting a good performing model is still challenging w.r.t hyper parameters, tuning etc..

**Comment by Suchi97** [comment]
Thanks @SachidanandAlle for your reply.
